src/test/java/com/github/cameltooling/lsp/internal/completion/CamelKubernetesServicesCompletionTest.java

```diff
@@ -39,13 +44,19 @@
 @EnableKubernetesMockClient(crud = true)
 class CamelKubernetesServicesCompletionTest extends AbstractCamelLanguageServerTest {
 
+    @Inject
 	private KubernetesClient client;
-	
+
+    @BeforeEach
```

Comment:
cleaning the cluster sounds more something for an @AfterEach so that it is not interfering with other tests